go.mod

```diff
@@ -48,24 +47,23 @@ require (
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/mattn/go-zglob v0.0.1 // indirect
 	github.com/miekg/coredns v0.0.0-20170910182647-1b60688dc8f7 // indirect
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
+	github.com/onsi/ginkgo v1.10.1
```

Comment:
Why are these dependencies being bumped in this PR?

Reply:
oh it's transitive dependencies

golangci-lint -> gosec -> github.com/onsi/ginkgo@v1.10.1